kwanj-k/ride-my-way
# Ft profile #158474084

### What does this PR do?
Creates a page/pages to display user profile and list of rides associated to the user

### Description of Task to be completed
* Create a user profile page 
* Display number of rides associated to user on the profille
* Create a page that displays a list of rides associated to the user

### How should I test it?
 * Clone this repository to a local folder
 * Navigate to the cloned repo
 * git checkout ft-request-options-#158474084
 * Open index.html using your favorite browser
 * Then click on the Your Profile to see profile
 * On the profile click on My-rides to see the list of all user rides

### What PT stories?
[ft-profile-#158474084] https://www.pivotaltracker.com/story/show/158474084

## Screenshots
*Profile

![oie_vehfbcigjoac](https://user-images.githubusercontent.com/33450849/41627374-c4e45264-7428-11e8-8df6-53f348631864.png)

*Rides List
![oie_zsexjjzelmum](https://user-images.githubusercontent.com/33450849/41627697-011fab9c-742a-11e8-888a-c0437d76d2d2.png)
